packages/fxa-auth-server/scripts/cleanup-secondary-emails.mjs

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env node
+
+/*
+Args:
+  --execute               Perform deletions (default is dry-run)
+  --batch-size=N          Max rows per batch (default 500)
+  --sleep-sec=N           Seconds to sleep between batches (default 2)
+*/
+
+import fs from 'fs';
+import path from 'path';
+import { setTimeout as sleep } from 'timers/promises';
+import mysql from 'mysql2/promise';
+
+function parseArgs(argv) {
```

Comment:
I personally like using `program` for defining and parsing args. It's quite a bit cleaner and gives nice CLI help / output. There's quite a few scripts that do this, check out `packages/fxa-auth-server/scripts/prune-tokens.ts`